bandit/core/issue.py

```diff
@@ -134,6 +206,7 @@ def from_dict(self, data, with_code=True):
         self.code = data["code"]
         self.fname = data["filename"]
         self.severity = data["issue_severity"]
+        self.cwe = cwe_from_dict(data["issue_cwe"])
```

Comment:
Is this why bandit 1.7.3 fails to load baseline data from JSON created by older bandit releases?

> [manager]	WARNING	Failed to load baseline data: 'issue_cwe'